app/models.py

```diff
@@ -125,7 +125,7 @@ class User(db.Model):
         onupdate=datetime.datetime.utcnow,
     )
     _password = db.Column(db.String, index=False, unique=False, nullable=False)
-    mobile_number = db.Column(db.String, index=False, unique=False, nullable=True)
+    mobile_number = db.Column(db.String, index=False, unique=True, nullable=True)
```

Comment:
Let's leave this as `False` given the alternative approach suggested.

```suggestion
    mobile_number = db.Column(db.String, index=False, unique=False, nullable=True)
```